integrationTests/vm/iele/vmAgar_test.go

```diff
@@ -328,6 +332,40 @@ func TestAgarioContractJoinGameReward(t *testing.T) {
 	assert.Equal(t, computedBalance, balanceOfSC)
 }
 
+func TestAgarioInvalidCodeShouldNotGenerateAccount(t *testing.T) {
```

Comment:
rename test as it is not related to agar

Reply:
done and moved to vmDeploy_test